### PR TITLE
Adding Czech translation

### PR DESCRIPTION
**Pull request by rbukovansky**
Hi, there is Czech translation for your project. Please, kindly merge. Thank you.

### CODE REVIEW

File: Sources/SwiftDate/SwiftDate.bundle/cs.lproj/SwiftDate.strings [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+"colloquial_p_y"					=	"minulý rok";		// year,past,singular:		"last year"
+"colloquial_p_yy"					=	"%d";				// year,past,plural:		"2015"
+
+"colloquial_f_m"					=	"příští měsíc";		// month,future,singular:	"next month"
+"colloquial_f_mm"					=	"za %d měsíce";		// month,future,plural:		"in 3 months"
+"colloquial_p_m"					=	"minulý měsíc";		// month,past,singular:		"past month"
+"colloquial_p_mm"					=	"před %d měsíci";	// month,past,plural:		"3 months ago"
+
+"colloquial_f_w"					=	"příští týden";		// week,future,singular:	"next week"
+"colloquial_f_ww"					=	"za %ld týdny";		// week,future,plural:		"in 3 weeks"
```
Comment (rbukovansky) [inline Sources/SwiftDate/SwiftDate.bundle/cs.lproj/SwiftDate.strings]: Just out of curiosity, why there is %ld, when there is %d? Thanks.

**Comment by rbukovansky** [comment]
@malcommac Hi, could you please review and merge if it's OK? Thanks.

**Comment by malcommac** [comment]
We need to better understand this issue https://github.com/malcommac/SwiftDate/issues/317 before merging all the translation posted recently.

**Comment by rbukovansky** [comment]
@malcommac I see... I will try it on my end and report what it is returning for my setup.

But could you please answer my question in code-review comment? Thanks.

**Comment by malcommac** [comment]
I've changed cs.lproj to cs-CZ.lproj
https://github.com/malcommac/SwiftDate/commit/1b762cfe7aca80ac4e43ffc757b06855c29c6723

**Comment by rbukovansky** [comment]
@malcommac OK. Sounds good to me.

BTW How about that %ld and %d difference, if there is any? Thanks.

**Comment by malcommac** [comment]
it's a typo, %d is enough. Just fixed.

**Comment by rbukovansky** [comment]
I see. OK. Thanks.

**Pull request merged**